contrib/linux/README.md

```diff
@@ -0,0 +1,5 @@
+# cloud-init configuration
```

Comment:
`linux` isn't really descriptive name

Reply:
It can run on gce and aws so it doesn't fit either folders.